python/py-allennlp/Portfile

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        allenai allennlp a8c44ae
+version             0.9.1
+revision            0
+name                py-${github.project}
+categories-append   textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         An open-source NLP research library, built on PyTorch.
+
+long_description    An Apache 2.0 NLP research library, built on\
+                    PyTorch, for developing state-of-the-art deep\
+                    learning models on a wide variety of linguistic\
+                    tasks.
+
+homepage            http://www.allennlp.org/
+
+checksums           rmd160  4fc2ffd4f892838d93b32d5a0a7931cb14149c82 \
+                    sha256  45e429692c5bad66f4ffa8dd4e204e0814df7ea48f50e02e91845daff89ba596 \
+                    size    7784697
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
incorrect dependency type as it installs ```console_scripts```

Reply:
??? I no grok.

Please post the corrected code.

Reply:
I have explained this in some of your other PRs before, the idea is to learn from that for future submissions not to simply copy-paste review suggestion. I am sure you should be able to understand this by now and figure out what change is needed...

Reply:
Sorry, I’m not still sure what you’re referring to. Repetition is the mother of learning, and copy-and-paste is also efficient so some suggested code would be appreciated, especially in this huge complicated PR with zillions of little interdependent bits.

Reply:
I assume you want `depends_lib` here, so adding that.